### PR TITLE
Translation support

### PR DESCRIPTION
**Pull request by HotoCocoaco**
What's translated:

- Almost every text.
- HUD.
- Panel.

What's not translated:

- Tag, e.g. [VSH2]
- Error Log.

Please tell me if I am missing anything.

### CODE REVIEW

File: addons/sourcemod/scripting/modules/handler.sp [hunk 1]
```diff
@@ -414,17 +414,17 @@ public Action ManageOnBossTakeDamage(const BaseBoss victim, int& attacker, int&
 					if( g_vsh2.m_hCfg.Get("messages.stabbed", stabbed, len) ) {
 						PrintCenterText(victim.index, "%s", stabbed);
 					} else {
-						PrintCenterText(victim.index, "You Were Just Tickled!");
+						PrintCenterText(victim.index, "%t", "boss_gotbackstabbed");
```
Comment (JassJam) [inline addons/sourcemod/scripting/modules/handler.sp]: `"boss_gotbackstabbed"` should be `"boss_got_backstabbed"`

File: addons/sourcemod/scripting/modules/commands.sp [hunk 2]
```diff
@@ -19,7 +19,7 @@ public Action QueuePanelCmd(int client, int args)
 	if( !g_vsh2.m_hCvars.Enabled.BoolValue ) {
 		return Plugin_Continue;
 	} else if( !client ) {
-		CReplyToCommand(client, "{olive}[VSH 2]{default} You can only use this command ingame.");
+		CReplyToCommand(client, "{olive}[VSH 2]{default} %t", "Command is in-game only");
```
Comment (JassJam) [inline addons/sourcemod/scripting/modules/commands.sp:23]: Missing `"Command is in-game only"` key in translations, and its better to rename it to `"command_ingame_only"` for convention.

File: addons/sourcemod/scripting/modules/commands.sp [hunk 3]
```diff
@@ -85,11 +85,11 @@ public void TurnToZeroPanel(const int client)
...
 	char strPanel[128];
 	//SetGlobalTransTarget(client);
-	Format(strPanel, 128, "Are you sure you want to set your queue points to 0?");
+	Format(strPanel, 128, "%T", "queue_set_0_confirm", client);
 	panel.SetTitle(strPanel);
-	Format(strPanel, 128, "YES");
+	Format(strPanel, 128, "%T", "Yes", client);
 	panel.DrawItem(strPanel);
-	Format(strPanel, 128, "NO");
+	Format(strPanel, 128, "%T", "No", client);
```
Comment (01Pollux) [inline addons/sourcemod/scripting/modules/commands.sp:93]: Missing `"Yes"` and `"No"` keys in translations.

File: addons/sourcemod/scripting/modules/commands.sp [hunk 4]
```diff
@@ -202,8 +206,11 @@ public Action SetBossMenu(int client, int args)
 		return Plugin_Continue;
 
 	Menu bossmenu = new Menu(MenuHandler_PickBosses);
-	bossmenu.SetTitle("%T", "set_boss_menu_title", client);
-	bossmenu.AddItem("-1", "%T", "bossmenu_none", client);
+	char temp[MAX_PANEL_MSG];
+	Format(temp, sizeof(temp), "%T", "set_boss_menu_title", client);
+	bossmenu.SetTitle(temp);
+	Format(temp, sizeof(temp), "%T", "set_boss_menu_title", client);
```
Comment (JassJam) [inline addons/sourcemod/scripting/modules/commands.sp]: You mean `"bossmenu_none"`?

File: addons/sourcemod/scripting/modules/commands.sp (hunk 2 above)
Comment (HotoCocoaco) [reply addons/sourcemod/scripting/modules/commands.sp:23]: nah. It's in `common.phrases.txt` which sourcemod has it. 
Check vsh2.sp L451, 452.

**Pull request merged**